semeio/jobs/design2params/design2params.py

```diff
@@ -0,0 +1,173 @@
+import logging
+import sys
+
+import numpy as np
+import pandas as pd
+
+
+def _create_handler(formatter, stream, level):
```

Comment:
Should there be a separate log handler on a job basis? Maybe in a shared location or be the responsibility of `ert`? @markusdregi, @jondequinor, we had a discussion on this earlier, any thoughts?

Reply:
Yes , we should probably make a common one which we can use across the jobs . Lets discuss on monday !